index.js

```
@@ -0,0 +1,110 @@
+      command: "render",
+      info,
+      width,
+      height,
+      oversampling,
+    }
+    worker.postMessage(payload)
+  })
+}
+
+// --- Managing the 9-Panel Grid ---
+export let formulas = new Array(9)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'formulas' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)